setup.py

```diff
@@ -36,6 +36,8 @@
     "joblib",
     "more-itertools",
     "webcolors",
+    "requests",
+    "webbrowser"
```

Comment:
Hi @Meekohi. When it's time to retun to this, could you also update `requirements.txt`? I think this section of `setup.py` is just for requirements-minimum, i.e. not packages that we import late and want to keep optional. I suppose Michael needs to decide whether `requests` will be mandatory. Historically music21 has had users brand new to Python who only install music21.

Reply:
also, `webbrowser` is in the standard lib, so we don't need to call it out.

Reply:
Roger that -- I do plan on revisiting this PR, but we've gotten dragged onto other work for now.